src/mbgl/shader/raster_shader.hpp

```diff
@@ -5,11 +5,18 @@
 
 namespace mbgl {
 
+namespace gl {
+template <class> class VertexBuffer;
+} // namespace gl
+
+class RasterVertex;
+
 class RasterShader : public Shader {
 public:
     RasterShader(gl::Context&, Defines defines = None);
 
-    void bind(int8_t* offset) final;
+    void bind(const gl::VertexBuffer<RasterVertex>&,
+              const int8_t* offset);
```

Comment:
Can we rename `offset` to something like `begin` to avoid confusion of the offset within a buffer with the offset of an attribute within a vertex?